lib/matplotlib/ticker.py

```diff
@@ -801,7 +801,7 @@ class LogFormatter(Formatter):
     Format values for log axis.
     """
     def __init__(self, base=10.0, labelOnlyBase=False,
-                 label_pruning=False):
+                 minor_thresholds=(1, 0.4)):
```

Comment:
I don't like the name `minor_thresholds` as the LogFormatter are minor/major agnostic.